Improve UI page

domdinicola opened the pull request:
- allow to schedule the sync task through the ui
- updated template (extend django admin)

czue commented on data_api/ui/views.py:
```diff
@@ -31,3 +36,21 @@ def import_org_view(request):
     })
+
+
+class BasicTaskAPIView(APIView, metaclass=ABCMeta):
```
This requires Py3

czue reviewed (commented):
This looks really nice. 

However, I'm getting this error in the console when I load the page (it still loads, but thinking we should fix this?)

```
DEBUG Exception while resolving variable 'title' in template 'ui/import_org.html'.
Traceback (most recent call last):
  File "/home/czue/.virtualenvs/rapidpro-api-py3/lib/python3.5/site-packages/django/template/base.py", line 882, in _resolve_lookup
    current = current[bit]
  File "/home/czue/.virtualenvs/rapidpro-api-py3/lib/python3.5/site-packages/django/template/context.py", line 87, in __getitem__
    raise KeyError(key)
KeyError: 'title'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/czue/.virtualenvs/rapidpro-api-py3/lib/python3.5/site-packages/django/template/base.py", line 888, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'title'
```
Also, we either need to make this Py2 compatible or wait till we've rolled out Py3 before merging.

domdinicola commented:
let's wait for django2, I don't think we will have those warnings

domdinicola committed:
improve page

domdinicola commented:
![Screenshot_2019-08-06_10-24-20](https://user-images.githubusercontent.com/2621572/62548655-241a6d80-b835-11e9-9f03-a93d38f8df3f.png)

domdinicola commented:
@czue pr ready for review

czue reviewed (approved):
Code looks good. From a UX perspective it took me a while to figure out that the "sync latest data" button actually syncs the entire warehouse. I wonder if we could change the text a bit to make that more clear? Like "Sync Entire Warehouse" or something?

I think one of the problems is that these two things are only semi-related, so having them lumped together on the same page is a little funny. But I think we can roll with it for now.

domdinicola committed:
added full sync action
added messages for admin actions
changed label in management page

domdinicola commented:
@czue amended PR, my idea was to make the page a more generic admin/management page. What do you think about that?

- changed label
- admin actions: added messages
- admin actions: add full sync

czue reviewed (approved):
:+1: sounds good
